instrumentation-core/spring/build.gradle

```diff
@@ -0,0 +1,40 @@
+plugins {
+	id 'org.springframework.boot' version '2.3.0.RELEASE'
+	id 'io.spring.dependency-management' version '1.0.9.RELEASE'
+	id 'java'
+}
+
+group = 'io.opentelemetry.instrumentation'
+version = '0.0.1-SNAPSHOT'
+
+apply from: "$rootDir/gradle/java.gradle"
+
+sourceCompatibility = '8'
+
+configurations {
+	compileOnly {
+		extendsFrom annotationProcessor
+	}
+}
+
+repositories {
+	mavenCentral()
+}
+
```

Comment:
I don't think you need this now that you are applying the common "java.gradle" above
```suggestion
```